### PR TITLE
[SETUP] muifonts.h: Make Asian fonts borderless inside Asian

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

Improve usability. Preparation for FontLink.

JIRA issue: [CORE-9616](https://jira.reactos.org/browse/CORE-9616)

## Proposed changes

- Modify `base/setup/lib/muifonts.h`.
- Add East Asian font substitutes for Asian users.

### CODE REVIEW

File: base/setup/lib/muifonts.h [hunk 1]
```diff
@@ -126,6 +200,29 @@ MUI_SUBFONT HebrewFonts[] =
     { L"Times New Roman Greek,161",    L"Times New Roman,161" },
     { L"Times New Roman TUR,162",      L"Times New Roman,162" },
     { L"Tms Rmn",                L"Times New Roman" },
+    /* The beginning of FE Asian fonts */
```
Comment (HBelusca) [inline base/setup/lib/muifonts.h:203]: I wonder why these are necessary in non-asian lists, like hebrew, hindi, etc.

Comment (katahiromz) [reply base/setup/lib/muifonts.h:203]: FontLink framework will realize the mix of various fonts in one text. If the font is not available, FontLink won't work.

Comment (katahiromz) [reply base/setup/lib/muifonts.h:203]: Ok, I will make a separated list.

Comment (katahiromz) [reply base/setup/lib/muifonts.h:203]: ― By the power of FontLink, ...
Latin users will be able to see Asian characters.
Asian users will be able to see Latin accented characters.

Comment (katahiromz) [reply base/setup/lib/muifonts.h:203]: I understand you don't like Asian complex. But FontLink is the only one way to realize real international Windows operating system.

Comment (JoachimHenze) [reply base/setup/lib/muifonts.h:203]: Is fontlinking available in 2k3sp2?
And if yes, how does MS do it?

Comment (katahiromz) [reply base/setup/lib/muifonts.h]: English Win2k3 doesn't have Asian fonts and FontLink in default settings. There is additional package for Far East Asian.
Win2k3-English-doesnt-have-FontLink

Japanese WinXP has Chinese, Japanese, and Korean fonts and Font Link.
WinXP-Japanese-has-FontLink-and-Chinese-fonts

Chinese WinXP has Chinese, Japanese, and Korean fonts and Font Link.
WinXP-Chinese-has-FontLink-and-Japanese-fonts

Comment (katahiromz) [reply base/setup/lib/muifonts.h]: So, I think that, FontLink is initially provided for East Asian people.
English and Latin users don't want FontLink so much.

Comment (katahiromz) [reply base/setup/lib/muifonts.h]: Fixed.

**Comment by katahiromz** [comment]
I found it is not necessary. Abandoned.

**Pull request closed without merging**